includes/islandora_newspaper_batch.inc

```diff
@@ -490,9 +490,10 @@ class IslandoraNewspaperPageBatchObject extends IslandoraNewspaperFlatBatchObjec
     islandora_paged_content_set_relationship($rels_ext, FEDORA_RELS_EXT_URI, 'isMemberOf', $this->parentId);
     // Add content model relationship.
     $this->models = 'islandora:newspaperPageCModel';
+    // If one is set, both are.
     if (isset($this->preprocessorParameters['generate_ocr'])) {
       module_load_include('inc', 'islandora_ocr', 'includes/derivatives');
-      islandora_ocr_set_generate_ocr_rels_ext_statement($this, $this->preprocessorParameters['generate_ocr']);
+      islandora_ocr_set_generating_rels_ext_statements($this, (bool) $this->preprocessorParameters['generate_ocr'], (bool) $this->preprocessorParameters['generate_hocr']);
```

Comment:
This is not a "change stuff" request, just curiosity: Why is bool casting needed?

Reply:
It's not really 100% necessary I guess, just that it's possible for stuff like ... strings to come in from the Drush command (seen people do things like --flag=true where they really just have to do --flag). I dunno, I always feel nicer normalizing this stuff